File: fix_content.js
const fs = require('fs');
const path = require('path');

const files = [
  'src/app/search/page.tsx',
  'src/app/hotels/page.tsx',
  'src/app/activities/page.tsx',
  'src/app/itineraries/builder/page.tsx',
  'src/app/itineraries/[id]/page.tsx',
  'src/app/dashboard/page.tsx'
];

files.forEach(file => {
  const fullPath = path.join(process.cwd(), file);
  if (!fs.existsSync(fullPath)) return;
  
  let content = fs.readFileSync(fullPath, 'utf8');

  // Locations
  content = content.replace(/London \(LHR\)/g, 'Mumbai (BOM)');
  content = content.replace(/Tokyo \(HND\)/g, 'Delhi (DEL)');
  content = content.replace(/LHR/g, 'BOM');
  content = content.replace(/HND/g, 'DEL');
  content = content.replace(/London/g, 'Mumbai');
  content = content.replace(/Tokyo/g, 'Delhi');
  content = content.replace(/Kyoto, Japan/g, 'Jaipur, Rajasthan');
  content = content.replace(/Kyoto/g, 'Jaipur');
  content = content.replace(/Tuscany & Umbria/g, 'Kerala Backwaters');
  content = content.replace(/Florence/g, 'Kochi');
  content = content.replace(/Chianti Hills/g, 'Munnar Plantations');
  content = content.replace(/The Japanese Archipelago/g, 'The Golden Triangle');

  // Airlines
  content = content.replace(/Japan Airlines/g, 'Air India');
  content = content.replace(/British Airways/g, 'Vistara');
  content = content.replace(/Qatar Airways/g, 'Vistara Premium');
  content = content.replace(/Emirates Boutique/g, 'IndiGo Luxe');
  content = content.replace(/DOH/g, 'HYD');
  content = content.replace(/DXB/g, 'BLR');
  content = content.replace(/NRT/g, 'DEL');

  // Hotels
  content = content.replace(/Aman Kyoto/g, 'Taj Mahal Palace');
  content = content.replace(/The Ritz-Carlton, Kyoto/g, 'The Oberoi Amarvilas');
  content = content.replace(/Suiran, a Luxury Collection/g, 'Rambagh Palace');
  content = content.replace(/Ryokan stays/g, 'Heritage Palace stays');

  // Activities
  content = content.replace(/Private Zen Garden Tour & Tea Ceremony/g, 'Private Fort Tour & High Tea');
  content = content.replace(/Exclusive Gion Geisha Dinner/g, 'Exclusive Rajputana Royal Dinner');
  content = content.replace(/Arashiyama Bamboo Grove at Dawn/g, 'Amer Fort at Dawn');
  content = content.replace(/Masterclass: Sushi Crafting/g, 'Masterclass: Authentic Spice Blending');
  content = content.replace(/Ryokan/g, 'Palace');

  // Currencies - Replace $ and £ with ₹
  // e.g. $1,284 -> ₹12,840
  content = content.replace(/\$([0-9,]+)/g, (match, p1) => `₹${p1}0`);
  content = content.replace(/£([0-9,]+)/g, (match, p1) => `₹${p1}0`);

  fs.writeFileSync(fullPath, content);
});

console.log('Done');

File: fix_content.py
import os
import re

files = [
  'src/app/search/page.tsx',
  'src/app/hotels/page.tsx',
  'src/app/activities/page.tsx',
  'src/app/itineraries/builder/page.tsx',
  'src/app/itineraries/[id]/page.tsx',
  'src/app/dashboard/page.tsx'
]

replacements = {
    'London \(LHR\)': 'Mumbai (BOM)',
    'Tokyo \(HND\)': 'Delhi (DEL)',
    'LHR': 'BOM',
    'HND': 'DEL',
    'London': 'Mumbai',
    'Tokyo': 'Delhi',
    'Kyoto, Japan': 'Jaipur, Rajasthan',
    'Kyoto': 'Jaipur',
    'Tuscany & Umbria': 'Kerala Backwaters',
    'Florence': 'Kochi',
    'Chianti Hills': 'Munnar Plantations',
    'The Japanese Archipelago': 'The Golden Triangle',
    'Japan Airlines': 'Air India',
    'British Airways': 'Vistara',
    'Qatar Airways': 'Vistara Premium',
    'Emirates Boutique': 'IndiGo Luxe',
    'DOH': 'HYD',
    'DXB': 'BLR',
    'NRT': 'DEL',
    'Aman Kyoto': 'Taj Mahal Palace',
    'The Ritz-Carlton, Kyoto': 'The Oberoi Amarvilas',
    'Suiran, a Luxury Collection': 'Rambagh Palace',
    'Ryokan stays': 'Heritage Palace stays',
    'Private Zen Garden Tour & Tea Ceremony': 'Private Fort Tour & High Tea',
    'Exclusive Gion Geisha Dinner': 'Exclusive Rajputana Royal Dinner',
    'Arashiyama Bamboo Grove at Dawn': 'Amer Fort at Dawn',
    'Masterclass: Sushi Crafting': 'Masterclass: Authentic Spice Blending',
    'Ryokan': 'Palace'
}

for file_path in files:
    full_path = os.path.join(os.getcwd(), file_path)
    if not os.path.exists(full_path):
        continue
    
    with open(full_path, 'r', encoding='utf-8') as f:
        content = f.read()

    for old, new in replacements.items():
        content = re.sub(old, new, content)

    # Multiply prices to make it look like INR
    content = re.sub(r'\$([0-9,]+)', lambda m: '₹' + m.group(1) + '0', content)
    content = re.sub(r'£([0-9,]+)', lambda m: '₹' + m.group(1) + '0', content)
    
    with open(full_path, 'w', encoding='utf-8') as f:
        f.write(content)

print('Done')
files.append('src/app/page.tsx')
for file_path in files:
    full_path = os.path.join(os.getcwd(), file_path)
    if not os.path.exists(full_path):
        continue
    
    with open(full_path, 'r', encoding='utf-8') as f:
        content = f.read()

    for old, new in replacements.items():
        content = re.sub(old, new, content)

    content = re.sub(r'\$([0-9,]+)', lambda m: '₹' + m.group(1) + '0', content)
    content = re.sub(r'£([0-9,]+)', lambda m: '₹' + m.group(1) + '0', content)
    
    with open(full_path, 'w', encoding='utf-8') as f:
        f.write(content)

print('Done page.tsx')
